data/base-stealing.yaml

```diff
@@ -5926,11 +5926,4 @@
   pawnable: false
   room: 13721
   town: Horse Clan
-- exceptionally_max: 305
```

Comment:
Why was this removed?

Reply:
It is in a citizen only shop in shard and shouldn't really go in there, sorry for lumping it together with the other change